app/models/concerns/oregon_digital/generic_metadata.rb

```diff
@@ -719,7 +719,7 @@ module GenericMetadata
         { name: 'rights_note', is_controlled: false },
         { name: 'rights_holder', is_controlled: false },
         { name: 'copyright_claimant', is_controlled: false },
-        { name: 'access_restrictions', is_controlled: false },
+        { name: 'access_restrictions_label', is_controlled: false },
```

Comment:
I think `is_controlled:` should be true here.

Reply:
done